source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -16,15 +16,17 @@ namespace
 
     Duration computeMedianDuration(std::vector<Duration> durations)
     {
+        Duration result;
         std::sort(durations.begin(), durations.end());
         if(durations.size() % 2 == 0)
         {
-            return (durations.at(durations.size() / 2 - 1) + durations.at(durations.size() / 2)) / 2;
+            result = (durations.at(durations.size() / 2 - 1) + durations.at(durations.size() / 2)) / 2;
```

Comment:
No, just remove the else, you should never have wrongly initialized data, ref `Duration result`.

Strictly speaking the program state is wrong between line 19 and 23, that is not good if people not paying attention to that refactor this function, suddenly this function returns the uninitialized value. This is btw a reason to use `const` and/or `auto`, because then you cannot get into that mess. But in this situation just return and you have one variable less and less program state that can be wrong.